modules/andes-core/common/src/main/java/org/wso2/andes/transport/network/security/ssl/SSLUtil.java

```diff
@@ -163,19 +163,19 @@ public static KeyStore getInitializedKeyStore(String storePath, String storePass
         InputStream in = null;
         try
         {
-            File f = new File(storePath);
-            if (f.exists())
-            {
-                in = new FileInputStream(f);
-            }
-            else 
-            {
-                in = Thread.currentThread().getContextClassLoader().getResourceAsStream(storePath);
-            }
-            if (in == null)
-            {
-                throw new IOException("Unable to load keystore resource: " + storePath);
+
+            in = Thread.currentThread().getContextClassLoader().getResourceAsStream(storePath);
+
+            if (null == in) { // Resource was not found in the classpath. Try to load it from the file system.
+                File f = new File(storePath);
+                if (f.exists())
+                {
+                    in = new FileInputStream(f);
+                } else {
+                    throw new IOException("Unable to load keystore resource: " + storePath);
+                }
```

Comment:
This was changed because in case of reading from file, IOPermissions needed to be given to that file from Java Security Policy.